From b63cf3cb92679254fbe4d300784af944e06defc7 Mon Sep 17 00:00:00 2001
From: Saleem Abdulrasool <compnerd@compnerd.org>
Date: Sun, 7 Oct 2012 16:09:02 -0700
Subject: [PATCH 4/4] systemd: add systemd-vconsole-setup

systemd-vconsole-setup.service executes a systemd utility
(systemd-vconsole-setup) which is not included in the initramfs, but, the
systemd-vconsole-setup unit is included.  Include the necessary dependency as
well.
---
 modules.d/98systemd/module-setup.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules.d/98systemd/module-setup.sh b/modules.d/98systemd/module-setup.sh
index 5576792..e998cda 100755
--- a/modules.d/98systemd/module-setup.sh
+++ b/modules.d/98systemd/module-setup.sh
@@ -30,6 +30,7 @@ install() {
         $systemdutildir/systemd-udevd \
         $systemdutildir/systemd-journald \
         $systemdutildir/systemd-sysctl \
+        $systemdutildir/systemd-vconsole-setup \
         $systemdsystemunitdir/emergency.target \
         $systemdsystemunitdir/sysinit.target \
         $systemdsystemunitdir/basic.target \
-- 
1.7.12.2

